Ctor 457 new pack network juniper mseries netconf (#4416)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-firewalls-juniper-ssg-snmp.md

> import Tabs from '@theme/Tabs';

Reviewer: Juniper SSG SNMP?

Reviewer 2: SAAS-1970

import TabItem from '@theme/TabItem';

## Connector dependencies

The following monitoring connectors will be installed when you install the **Juniper SSG** connector through the

> **Configuration > Monitoring Connectors** menu:

Reviewer: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu:
```

> **Configuration > Monitoring Connectors** menu:

Reviewer 2: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu:
```

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias        | Service Template                           | Service Description                                                                                                                       | Discovery  |
|:---------------------|:-------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------|:----------:|

> | Traffic-Generic-Id   | Net-JuniperSSG-Traffic-Generic-Id-custom   | Check the bandwidth of the interface. For each checks the name of the interface will appear (« label » shortcut describing the interface) |            |

Reviewer: ```suggestion
| Traffic-Generic-Id   | Net-JuniperSSG-Traffic-Generic-Id-custom   | Check the bandwidth of the interface. For each check the name of the interface will appear (« label » shortcut describing the interface) |            |
```

> | Traffic-Generic-Id   | Net-JuniperSSG-Traffic-Generic-Id-custom   | Check the bandwidth of the interface. For each checks the name of the interface will appear (« label » shortcut describing the interface) |            |

Reviewer 2: ```suggestion
| Traffic-Generic-Id   | Net-JuniperSSG-Traffic-Generic-Id-custom   | Check the bandwidth of the interface. For each check the name of the interface will appear (« label » shortcut describing the interface) |            |
```

| Traffic-Generic-Name | Net-JuniperSSG-Traffic-Generic-Name-custom | Check the bandwidth of the interface. For each checks the name of the interface will appear (« label » shortcut describing the interface) |            |

> | Traffic-Global       | Net-JuniperSSG-Traffic-Global-custom       | Check the bandwidth of  interfaces. For each checks the name of the interface will appear (« label » shortcut describing the interface)   |            |

Reviewer: ```suggestion
| Traffic-Global       | Net-JuniperSSG-Traffic-Global-custom       | Check the bandwidth of  interfaces. For each check the name of the interface will appear (« label » shortcut describing the interface)   |            |
```

> | Traffic-Global       | Net-JuniperSSG-Traffic-Global-custom       | Check the bandwidth of  interfaces. For each checks the name of the interface will appear (« label » shortcut describing the interface)   |            |

Reviewer 2: ```suggestion
| Traffic-Global       | Net-JuniperSSG-Traffic-Global-custom       | Check the bandwidth of  interfaces. For each check the name of the interface will appear (« label » shortcut describing the interface)   |            |
```

#### Host discovery

| Rule name       | Description                                                                                                                                                                                                                                     |
|:----------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **Net-Juniper-SSG-custom** host template. |

More information about discovering hosts automatically is available on the [dedicated page](/docs/monitoring/discovery/hosts-discovery).

#### Service discovery

| Rule name                | Description |
|:-------------------------|:------------|

> | Net-Juniper-SSG-Vpn-Name |             |

Reviewer 2: ```suggestion
| Net-Juniper-SSG-Vpn-Name | Discover VPN and monitor them with the Vpn-Status service template |
```

### Pack

The installation procedures for monitoring connectors are slightly different depending on [whether your license is offline or online](../getting-started/how-to-guides/connectors-licenses.md).

1. If the platform uses an *online* license, you can skip the package installation
instruction below as it is not required to have the connector displayed within the

> **Configuration > Monitoring Connectors** menu.

Reviewer: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu.
```

> **Configuration > Monitoring Connectors** menu.

Reviewer 2: ```suggestion
*Configuration > Connectors > Monitoring Connectors** menu.
```

</Tabs>

2. Whatever the license type (*online* or *offline*), install the **Juniper SSG** connector through

> the **Configuration > Monitoring Connectors** menu.

Reviewer: ```suggestion
the **Configuration > Connectors > Monitoring Connectors** menu.
```

> the **Configuration > Monitoring Connectors** menu.

Reviewer 2: ```suggestion
the **Configuration > Connectors > Monitoring Connectors** menu.
```

| Macro    | Description                                     | Default value     | Mandatory   |
|:---------|:------------------------------------------------|:------------------|:-----------:|

> | CRITICAL | Critical threshold in percent (1min,5min,15min) | 90,85,80          |             |

Reviewer: keep the old text

> | WARNING  | Warning threshold in percent (1min,5min,15min)  | 80,75,70          |             |

Reviewer 2: ```suggestion
| CRITICAL | Critical threshold in percent for the average value during the last minute, the last 5 minutes and the last 15 minutes. | 90,85,80          |             |
| WARNING  | Warning threshold in percent for the average value during the last minute, the last 5 minutes and the last 15 minutes.  | 80,75,70          |             |
```

| EXTRAOPTIONS   | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                                                                                              | --verbose         |             |

</TabItem>
<TabItem value="Vpn-Status" label="Vpn-Status">

| Macro              | Description                                                                                                                                                                                           | Default value                                              | Mandatory   |
|:-------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------------------|:-----------:|

> | FILTERNAME         | Filter VPN name (can be a regexp)                                                                                                                                                                     | .*                                                         |             |

Reviewer: keep old text

> | FILTERNAME         | Filter VPN name (can be a regexp)                                                                                                                                                                     | .*                                                         |             |

Reviewer 2: ```suggestion
| FILTERNAME         | Check only VPNs whose name match the given regular expression                                                                                                                                                                    | .*                                                         |             |
```

| CRITICALUPDATETIME | Critical threshold for update time (in seconds)                                                                                                                                                       |                                                            |             |
| EXTRAOPTIONS       | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                                                                    | --verbose                                                  |             |

</TabItem>
<TabItem value="Vpn-Usage" label="Vpn-Usage">

| Macro              | Description                                                                                        | Default value     | Mandatory   |
|:-------------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTERNAME         | Filter VPN name (can be a regexp)                                                                  | .*                |             |

Reviewer 2: ```suggestion
| FILTERNAME         | Check only VPNs whose name match the given regular expression                                                                  | .*                |             |
```
